conf.json

```diff
@@ -0,0 +1,23 @@
+{
+    "dev": {
+        "host": "127.0.0.1",
+        "port": 0,
+        "db": {
+            "host": "localhost",
+            "port": 0,
+            "user": "",
+            "password": ""
+        }
+    },
```

Comment:
In #19 we decided to scrap the `conf.json` files and use `.env` instead. see [.env.example](https://github.com/vlad-iancu/proiect-ip/blob/main/.env.example)